.mergify.yml

```diff
@@ -1,9 +1,6 @@
 queue_rules:
   - name: automerge
     conditions:
-      - check-success=python3.8-x64 windows-latest
```

Comment:
I need to push a PR to update this config with GitHub merge queues we don't need this, and we can only use mergify for automated backporting.

Reply:
Ditto. I actually quite like the GitHub queues, we could move some more expensive checks to the final merge instead of running them each time